stytch/b2b/session.go

```diff
@@ -42,6 +45,15 @@ type SessionRevokeResponse struct {
 	StatusCode int    `json:"status_code,omitempty"`
 }
 
+type SessionsGetJWKSParams struct {
+	ProjectID string `json:"project_id"`
+}
+
+type SessionsGetJWKSResponse struct {
+	RequestID string       `json:"request_id,omitempty"`
+	Keys      []shared.Key `json:"keys,omitempty"`
+}
```

Comment:
Similarly here, I think we might want the same alias
```suggestion
type SessionsGetJWKSResponse struct {
	RequestID string       `json:"request_id,omitempty"`
	Keys      []Key `json:"keys,omitempty"`
}

type Key = shared.Key
```